### DISCUSSION TITLE
Invalid VIEWONTAG_PATCH

### DISCUSSION THREAD

**Discussion by kimuxer** [bakkeby/dwm-flexipatch#332] [Q&A]
VIEWONTAG_PATCH doesn't seem to work,

I took a look at the original patch
`+static const Bool viewontag         = True;     /* Switch view on tag switch */`
But in flexipatch's config.def.h, there is nothing related to VIEWONTAG_PATCH

In original patch , `dwm.c` :
```
+		if(viewontag && ((arg->ui & TAGMASK) != TAGMASK))
+			view(arg);
```
flexipatch, `dwm.c`:
```
#if VIEWONTAG_PATCH
	if ((arg->ui & TAGMASK) != selmon->tagset[selmon->seltags])
		view(arg);
#endif // VIEWONTAG_PATCH
```

Is this a bug? Or was there a modification somewhere else but I didn't notice

**Comment by bakkeby** [comment]
Yes I did not include this. Adding an additional toggle in the config didn't seem to make much sense when you control the inclusion via `patches.h` anyway. I think it is a bit redundant for the original patch to include that, but that is just my opinion.

**Comment by kimuxer** [comment]
Agree.
 I'll remove it from my patch list
I'm doing a subtraction on my patch list
Thanks again for your project which saved me a ton of time and energy
